src/node.js

```diff
@@ -49,7 +49,7 @@ export default class NodeSourceMap extends SourceMap {
     lineOffset = 0,
   }: GenerateEmptyMapOptions): NodeSourceMap {
     let map = new NodeSourceMap(projectRoot);
-    map.addEmptyMap(relatifyPath(sourceName, projectRoot), sourceContent, lineOffset);
```

Comment:
Isn't `map.addEmptyMap` calling `addEmptyMap` in `src/SourceMap.js` which also fixes that path?

Reply:
Yes? I just moved the `relatifyPath` from this static function to the `addEmptyMap` that it calls. So when calling `map.addEmptyMap` it behaves as expected

Reply:
🙈  Is temporary colorblindness a thing?